tutorials/3d/fps_tutorial/part_six.rst

```diff
@@ -256,25 +256,25 @@ Let's add the code needed to fill these :ref:`Labels <class_Label>`. Select ``De
     extends Control
 
     func _ready():
-        $OS_Label.text = "OS:" + OS.get_name()
-        $Engine_Label.text = "Godot version:" + Engine.get_version_info()["string"]
+        $OS_Label.text = "OS: " + OS.get_name()
+        $Engine_Label.text = "Godot version: " + Engine.get_version_info()["string"]
 
     func _process(delta):
-        $FPS_Label.text = "FPS:" + str(Engine.get_frames_per_second())
+        $FPS_Label.text = "FPS: " + str(Engine.get_frames_per_second())
 
 Let's go over what this script does.
 
 ______
 
-In ``_ready`` we set the ``OS_Label``'s text to the name provided by :ref:`OS <class_OS>` using the ``get_name`` function. This will return the
-name of the OS (or Operating System) that Godot was compiled for. For example, when you are running Windows it will return ``Windows``, while when you
-are running Linux it will return ``X11``.
+In ``_ready``, we set the ``OS_Label``'s text to the name provided by :ref:`OS <class_OS>` using the ``get_name`` function. This will return the
+name of the OS (or Operating System) for which Godot was compiled. For example, when you are running Windows, it will return ``Windows``, while when you
+are running GNU/Linux, it will return ``X11``.
```

Comment:
Nope. We use "Linux" as the name of the OS as a whole, for very good reasons. Don't try to argue with it, I've wasted countless hours on this topic already and have no wish to continue.

The Linux operating system is way more than just the GNU tooling and the Linux kernel. If you want a technically descriptive name, you'd need to name it GNU/Linux/[systemd,sysvinit]/[Mesa,nvidia]/[KDE,GNOME,Xfce,Cinnamon,...]/[Ubuntu,Fedora,Debian,...]/FreeType/etc.

Reply:
I appreciate your proofreading but the most important quality of documentation is *consistency*. So don't make such arbitrary changes in one page, it would decrease the docs' overall quality.
```
$ rg "GNU/Linux" | wc -l
0
$ rg "[^GNU/]Linux" | wc -l
58
```

Reply:
> Nope. We use "Linux" as the name of the OS as a whole, for very good reasons. Don't try to argue with it, I've wasted countless hours on this topic already and have no wish to continue.
> 
> The Linux operating system is way more than just the GNU tooling and the Linux kernel. If you want a technically descriptive name, you'd need to name it GNU/Linux/[systemd,sysvinit]/[Mesa,nvidia]/[X11,Wayland,Mir]/[KDE,GNOME,Xfce,Cinnamon,...]/[Ubuntu,Fedora,Debian,...]/FreeType/etc.

@akien-mga : [The Unix-Haters Handbook](https://en.wikipedia.org/wiki/The_Unix-Haters_Handbook) : "[...] What  is  this  horrible  fascination  with  Unix?  The  operating  system  of  the 1960s,  still  gaining  in  popularity  in  the  1990s.  A  horrible  system,  except that all the other commercial offerings are even worse. The only operating system that is so bad that people spend literally millions of dollars trying to improve  it.  Make  it  graphical  (now  that’s   **an oxymoron,  a  graphical  user interface for Unix**) [...]."

"Development of the [GNU operating system](https://en.wikipedia.org/wiki/GNU) was initiated by Richard Stallman while he worked at MIT Artificial Intelligence Laboratory. It was called the GNU Project, and was publicly announced on September 27, **1983** [...]."

"The [Linux kernel](https://en.wikipedia.org/wiki/Linux_kernel) was conceived and created in **1991** by Linus Torvalds [...]."

I am **NOT** trying to convince you to go back on your decision. What I am doing is providing logical and historical arguments (not "arguing" = emotional quarrel) to prove that my alterations have been justified.

Reply:
Don't worry, I've been using Linux-based operating systems and GNU software for 15 years, I have my own opinion and I know the history of all involved projects.